manifests/candlepin.pp

```diff
@@ -78,6 +78,27 @@
   $client_key = "${pki_dir}/private/${java_client_cert_name}.key"
 
   if $deploy {
+    privkey { $ca_key:
+      key_pair      => $default_ca,
+      unprotect     => true,
```

Comment:
Would it make sense to enhance https://github.com/theforeman/puppet-certs/blob/master/manifests/keypair.pp with these parameters to reduce duplication?

The alternative is to pass in `candlepin::ca_key_password`